server/src/main/java/io/pravega/schemaregistry/storage/client/WireCommandClient.java

```diff
@@ -108,7 +106,7 @@
                     RawClient connection = new RawClient(ModelHelper.encode(uri), connectionPool);
                     final long requestId = connection.getFlow().asLong();
 
-                    return sendRequest(connection, requestId, new WireCommands.CreateTableSegment(requestId, tableName, false, delegationToken))
+                    return sendRequest(connection, requestId, new WireCommands.CreateTableSegment(requestId, tableName, false, 0, delegationToken))
```

Comment:
This whole class seems basically a copy + extension of the `SegmentHelper` class in the Pravega Controller: https://github.com/pravega/pravega/blob/master/controller/src/main/java/io/pravega/controller/server/SegmentHelper.java
This PR itself is a maintenance cost associated to that fact. Maybe is not the time for this change now, but it may be interesting to think about putting `SegmentHelper` class in Pravega in the "shared" package, for instance, and import that class here from that package. If we would have done that, this PR would not be necessary, as the update for the new KVT API was done long ago. If not, maybe using the KVT Client as @andreipaduroiu suggested would make this simpler, right? Why do we need to send these low level commands directly if there there is a client in place?

Reply:
A new issue has been created and the changes will be done as part of it's solution. Issue is:- https://github.com/pravega/schema-registry/issues/222. As part of that issue, all changes will be taken to remove use of existing low level wire commands and use KVT client instead. This PR fixes the existing behavior of Schema Registry to use latest Pravega.

Reply:
@shshashwat please, add a TODO in the code at the point where the test is deleted so you remember to add it later (btw, these are the guidelines to add TODOs in the code: https://github.com/pravega/pravega/wiki/Contributing#todos-in-the-code)

Reply:
Added the TODO, thanks for the idea.